NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/CountriesConsentActivity.cs

```diff
@@ -26,7 +26,8 @@ public class CountriesConsentActivity: AppCompatActivity
         private TextView _notificationText;
         private TextView _beAwareText;
         private TextView _consentExplanationText;
-        private Button _consentButton;
+        private Button _consentWithEUButton;
```

Comment:
Change to "consentButton_EU"

Reply:
Went with _consentButtonEU to follow convention